pkg/aspect/reportManager.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2017 Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aspect
+
+import (
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	rpc "github.com/googleapis/googleapis/google/rpc"
+
+	"istio.io/mixer/pkg/adapter"
+	config2 "istio.io/mixer/pkg/adapter/config"
```

Comment:
instead of `config2`, how about `acfg`, `aconfig`, `adptr`, or `newcfg` (with a TODO to fix) ?

also, i think we need a package comment on `//pkg/adapter/config`.